test/integration/dashboard_test.rb

```diff
@@ -9,7 +9,9 @@ def setup
   def assert_dashboard_link(text)
     visit dashboard_path
     assert page.has_link?(text), "link '#{text}' was expected, but it does not exist"
-    click_link(text)
+    within ".ui-helper-hidden" do
```

Comment:
We shouldn't test that users can click hidden links, it should probably test the links within `widget status-table`?